src/com/facebook/buck/swift/SwiftBuckConfig.java

```diff
@@ -247,4 +249,18 @@ public boolean getAddXctestImportPaths() {
   public boolean getIndexWhileBuilding() {
     return delegate.getBooleanValue(SECTION_NAME, INDEXING_WHILE_BUILDING, false);
   }
+
+  /**
+   * If true, the index store generated by swift compiler will be global (module independent) and non-cacheable
+   */
+  public boolean getLocalIndexStore() {
+    return delegate.getBooleanValue(SECTION_NAME, LOCAL_INDEX_STORE, false);
+  }
+
+  /**
+   * If present, Enables Swift Compiler multi-threading and specify number of threads
+   */
+  public Optional<String> getNumberOfThreads() {
+    return delegate.getRawValue(SECTION_NAME, NUMBER_OF_THREADS);
+  }
```

Comment:
I'm not necessarily against this config, but as we discussed earlier, I don't see it solving our current problem in regards to `-num-threads 12`: we have people building with macs that have 8, 12 e 16 threads currently. How can we make this value dynamic for each user automatically?

My suggestion is to fetch the number of threads Buck is using - it does the math somewhere and we could use the same value. If this turns out to be very cumbersome to get (don't really feel like adding a lot of changes for this) then this approach may be ok, we just need to think about how can we personalize this config for each dev automatically in our CLI.